UnitsNet/IArithmeticQuantity.cs

```diff
@@ -1,20 +1,60 @@
-// Licensed under MIT No Attribution, see LICENSE file at the root.
+﻿// Licensed under MIT No Attribution, see LICENSE file at the root.
 // Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
 
-using System;
+#if NET
 using System.Numerics;
+#endif
 
 namespace UnitsNet;
 
 /// <summary>
-///     An <see cref="IQuantity{TSelf, TUnitType}"/> that (in .NET 7+) implements generic math interfaces for arithmetic operations.
+///     Represents a quantity that has both magnitude and direction, supporting various arithmetic operations and
+///     comparisons.
+/// </summary>
+/// <remarks>
+///     This interface defines standard linear arithmetic operations such as addition, subtraction, multiplication, and
+///     division.
+///     These types of quantities naturally support comparison operations with either absolute or relative tolerance, which
+///     is useful for determining equality within a certain margin of error.
+///     <para>
+///         For more information, see the Wikipedia page on
+///         <a href="https://en.wikipedia.org/wiki/Dimensional_analysis#Geometry:_position_vs._displacement">
+///             Dimensional
+///             Analysis
+///         </a>
+///         .
+///     </para>
+/// </remarks>
+/// <typeparam name="TSelf">The type that implements this interface.</typeparam>
+public interface ILinearQuantity<TSelf> : IQuantityInstance<TSelf>
+#if NET7_0_OR_GREATER
+    , IAdditiveIdentity<TSelf, TSelf>
+#endif
+    where TSelf : ILinearQuantity<TSelf>
+{
+#if NET7_0_OR_GREATER
+    /// <summary>
+    ///     The zero value of this quantity.
+    /// </summary>
+    static abstract TSelf Zero { get; }
+
+    static TSelf IAdditiveIdentity<TSelf, TSelf>.AdditiveIdentity
+    {
+        get => TSelf.Zero;
+    }
+
+#endif
+}
+
+/// <summary>
+///     An <see cref="IQuantity{TSelf, TUnitType}" /> that (in .NET 7+) implements generic math interfaces for arithmetic
+///     operations.
 /// </summary>
 /// <typeparam name="TSelf">The type itself, for the CRT pattern.</typeparam>
 /// <typeparam name="TUnitType">The underlying unit enum type.</typeparam>
-public interface IArithmeticQuantity<TSelf, TUnitType> : IQuantity<TSelf, TUnitType>
+public interface IArithmeticQuantity<TSelf, TUnitType> : IQuantity<TSelf, TUnitType>, ILinearQuantity<TSelf>
```

Comment:
I think it'll be more to clear to name it `ILinearArithmeticQuantity`, if it's arithmetic is for linear quantities.

Reply:
Do you mean to rename the `IArithmeticQuantity` interface (which would be a breaking change) or the `ILinearQuantity` interface?

Reply:
Gotcha, no I meant renaming `IArithmeticQuantity` to `ILinearArithmeticQuantity` since it's now scoped to only linear quantities, where as before it covered all quantities.

Or is it just that we haven't implemented for affine/logarithmic yet, but this may come later?